dissect/hypervisor/disk/vdi.py

```diff
@@ -67,18 +67,12 @@ def _read(self, offset, length):
 
 
 class Vbox:
```

Comment:
Do you think it's worth moving this class to `dissect/hypervisor/descriptor/vbox.py` with this MR? Would fit in better with the rest there.

Reply:
Yes, that makes sense, good idea.